src/fake_bpy_module/transformer/data_type_refiner.py

```diff
@@ -76,6 +79,19 @@
 _REGEX_DATA_TYPE_STARTS_WITH_COLLECTION = re.compile(r"^(list|tuple|dict)")
 
 
+def get_rna_enum_items(dtype_str: str) -> str:
+    rna_enum_name = dtype_str.split("`")[1][len("rna_enum_"):]
+    rna_enum_path = (Path(config.get_input_dir())
+                     / "bpy_types_enum_items"
+                     / f"{rna_enum_name}.rst")
+    content = rna_enum_path.read_text()
+    return ', '.join(
+        f'"{v.split(":")[1]}"'
+        for v in content.splitlines()
+        if v.startswith(":")
+    )
```

Comment:
We use docutils package to parse .rst file safely.
Is it possible to use publish_doctree like [mod_aplier](https://github.com/nutti/fake-bpy-module/blob/ae6853ef24e8d311df4865f26005e40633f80fff/src/fake_bpy_module/transformer/mod_applier.py#L273) do this?

Reply:
Done